cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -429,4 +434,24 @@ export class Application {
         doesExistText("Upload a local binary", rbacRules["Upload binary"]);
         clickByText(button, "Cancel");
     }
+
+    validateIssues(appIssues: AppIssue[]): void {
+        Issues.openSingleApplication(this.name);
+        appIssues.forEach((currentIssue) => {
+            cy.contains(currentIssue["name"])
+                .closest(trTag)
+                .within(() => {
+                    validateTextPresence(singleAppLabels.issue, currentIssue["name"]);
+                    validateTextPresence(singleAppLabels.category, currentIssue["category"]);
+                    validateTextPresence(singleAppLabels.source, currentIssue["source"]);
+                    cy.get(singleAppLabels.target).within(() => {
+                        currentIssue["target"].forEach((currentTarget) => {
+                            validateTextPresence(liTag, currentTarget);
+                        });
+                    });
+                    validateNumberPresence(singleAppLabels.effort, currentIssue["effort"]);
+                    validateNumberPresence(singleAppLabels.files, currentIssue["affectedFiles"]);
```

Comment:
Is there a reason for using the array syntax only in this function?

`currentIssue["name"]` , `currentIssue["category"]`, etc could be just `currentIssue.name`, `currentIssue.category`,  etc

Reply:
Done